pkgs/servers/jellyseerr/package.json

```diff
@@ -89,42 +89,41 @@
     "secure-random-password": "0.2.3",
     "semver": "7.3.8",
     "sqlite3": "5.1.4",
-    "swagger-ui-express": "4.6.0",
-    "swr": "2.0.0",
-    "typeorm": "0.3.11",
+    "swagger-ui-express": "4.6.2",
+    "swr": "2.0.4",
+    "typeorm": "0.3.12",
```

Comment:
reading the release notes it states:
`Before you yarn build please downgrade your typeorm version by yarn add typeorm@0.3.11`

Reply:
Right above that it says "Warning (For windows users)" so this shouldn't be of concern, right?

Reply:
sorry, you are right.